cmd/notation/key.go

```diff
@@ -186,14 +187,12 @@ func addExternalKey(ctx context.Context, opts *keyAddOpts, pluginName, keyName s
 	if id == "" {
 		return config.KeySuite{}, errors.New("missing key id")
 	}
-	mgr := manager.New()
-	p, err := mgr.Get(ctx, pluginName)
+	mgr := plugin.NewCLIManager(dir.PluginFS())
+	// Check existence of plugin with name pluginName
+	_, err := mgr.Get(ctx, pluginName)
```

Comment:
Accroding to https://github.com/notaryproject/notation-go/pull/207
We should get the plugin and and call GetMetadata() to check is it a valid plugin.

Reply:
This is only adding the external key. We will validate the plugin when we use it. See comment below.